tests/test_hep_eptp_vllm_tp.py

```diff
@@ -195,20 +195,21 @@ def test_hep_eptp_vllm_tp_dst_ep1_tp2_pp1_src_ep4_tp2_pp1():
     assert param_sync_group.ep_num_mapping == tuples[0] / tuples[3]
     assert param_sync_group.tp_num_mapping == tuples[1] // tuples[4]
 
-    # Judge allgather actors
-    allgather_actors = param_sync_group.send_actors_to_allgather_routed_experts
+    # Judge alltoall actors
+    alltoall_actors = param_sync_group.send_actors_to_alltoall_routed_experts
```

Comment:
It might be better to leave 1 or 2 UTs to check if allgather actors are correctly detected and grouped.

Reply:
For tp-equal cases, alltoall is enough.